server/game_service.py

```diff
@@ -37,12 +37,12 @@ class GameService(Service):
     """
 
     def __init__(
-        self,
-        database: FAFDatabase,
-        player_service,
-        game_stats_service,
-        rating_service: RatingService,
-        message_queue_service: MessageQueueService
+            self,
+            database: FAFDatabase,
+            player_service,
+            game_stats_service,
+            rating_service: RatingService,
+            message_queue_service: MessageQueueService
```

Comment:
Please fix these. This is not how we are formatting in this repo. We use only 1 level of indentation.

Reply:
Then why did nobody set it that way in intellij code style?

Reply:
We don't use intellij

Reply:
I meant Pycharm but yeah. Also making an .editorconfig would do the trick